lib/scope.js

```diff
@@ -2106,35 +2108,126 @@ module.exports = {
     // TODO: Maybe we should return something else or something in addition about used vars
   },  // Ends scope.setFields()
 
-  // TODO: Move this to under `normalizeTable`
+  increment_row_use: async function ( scope, { row }) {
+    /** Accumulate number of times this row was used. Esp important for `.target_number`.
+    *   Mutates `row` and its props. */
+
+    // Increment the number of times a row has been used in total.
+    row.times_used += 1;
+    if ( !row.used_for ) { row.used_for = {}; }
+    if ( row.used_for[ row.var ] ) {
+      row.used_for[ row.var ] += 1;
+    } else {
+      row.used_for[ row.var ] = 1;
+    }
+
+    if ( row.source ) {
+      // Increment the number times the source row's artificial row trackers
+      // were used. For example, trackers like those for `.there_are_any` and
+      // `.there_is_another` (for `.target_number` rows)
+      if ( row.source.used_for[ row.var ] ) {
+        row.source.used_for[ row.var ] += 1;
+      } else {
+        row.source.used_for[ row.var ] = 1;
+      }
+      // Increment the number of times the source row has been used in total.
+      row.source.times_used += 1;
+
+      // These are important for rows like `target_number` rows that are
+      // accumulators for the artificial rows they support.
+    }
+  },  // Ends scope.increment_row_use()
+
+
   ensureSpecialRows: async function ( scope, { var_data, from_story_table=true }) {
     /* Given a list of variable data objects, add more objects or mutate
-    * rows under special circumstances:
-    * - `.target_number` may indicate the need for a `.there_is_another` row.
-    * - `.there_is_another` in a table is invalid
+    *    row objects under special circumstances:
+    * - `.target_number` will create `.there_are_any` and `.there_is_another` rows.
+    *    If they're unused in the end, that's fine.
+    * - `.there_is_another` with a value of True in a Story Table is invalid and
+    *    will be neutralized.
+    * 
+    * To learn the bigger picture, see the docs folder for info about Story Tables.
```

Comment:
```suggestion
    * To learn the bigger picture, see "docs/story_tables.md"
```

A link would be nice, but idk how to get that working, either in GitHub or locally